enhancements/20231113-witness-node-support.md

```diff
@@ -0,0 +1,72 @@
+# Support witness node
+
+## Summary
+
+The witness node is a lightweight node that only runs the control plane components. The witness node is not schedulable and does not run any workload. The witness node forms a quorum with the other two nodes.
+
+### Related Issues
+
+https://github.com/harvester/harvester/issues/3266
+
+## Motivation
+
+### Goals
+
+Create a three-node cluster with one witness node.
+
+## Proposal
+
+### User Stories
+
+For the edge use case, the witness node support benefits both cost and high availability. The classical scenario would be two nodes and one witness node.
```

Comment:
For user stories, they need to defined, more from user pespective, e.g.
(1)  User can install new witness node
How is this node installed, resource requrirements;  detailed steps
(2)  User can remove witness node
How is this node removed,
(3) User can/cannot change witness node role
how/why
...

Reply:
I will update the user story once the installation steps are clear.